$MirOS: ports/devel/gmp/patches/patch-configure_in,v 1.1 2006/01/25 13:54:48 tg Exp $
$OpenBSD: patch-configure,v 1.4 2003/04/18 01:22:53 naddy Exp $
--- configure.in.orig	2004-09-21 13:02:43.000000000 +0000
+++ configure.in	2008-08-08 18:19:09.000000000 +0000
@@ -1652,7 +1652,7 @@ fi
 # Using the compiler is a lot easier than figuring out how to invoke the
 # assembler directly.
 #
-test -n "$CCAS" || CCAS="$CC -c"
+test -n "$CCAS" || CCAS="$CC -D_ASM_SOURCE -c"
 AC_SUBST(CCAS)
 
 case $host in
@@ -2040,11 +2040,6 @@ if test "$gmp_asm_syntax_testing" != no;
       if test "$enable_profiling" != no; then
         GMP_ASM_X86_MCOUNT
       fi
-      case $host in
-        *-*-openbsd*)
-          GMP_DEFINE(_GLOBAL_OFFSET_TABLE_,__GLOBAL_OFFSET_TABLE_)
-          ;;
-      esac
       ;;
   esac
 fi
